vendored/sqlite3-parser/src/parser/ast/mod.rs

```diff
@@ -1622,6 +1622,8 @@ pub enum PragmaName {
     LegacyFileFormat,
     /// Return the total number of pages in the database file.
     PageCount,
+    // `page_size` pragma
```

Comment:
This should have `///` and read something like `/// Return the page size of the database in bytes.`